tests/Stripe/PlanTest.php

```diff
@@ -43,6 +43,58 @@ public function testIsCreatable()
         $this->assertInstanceOf("Stripe\\Plan", $resource);
     }
 
+    public function testIsCreatableMetered()
```

Comment:
I'm not sure these tests are needed -- there is already a test for the `create` method and the library itself doesn't care about the request parameters, so the only thing being tested here is that the parameters provided in the tests pass stripe-mock's validation.

@brandur-stripe wdyt?

Reply:
I'd rather not have the tests then, to keep things consistent in the test suite.